graf2d/primitivesv7/inc/ROOT/RLine.hxx

```diff
@@ -26,9 +26,9 @@ namespace Experimental {
 
 class RLine : public RDrawable {
 
-   RPadPos fP1;                            ///< line begin
-   RPadPos fP2;                            ///< line end
-   RAttrLine  fAttrLine{this, "line_"};    ///<! line attributes
+   RPadPos    fP1;                        ///< line begin
+   RPadPos    fP2;                        ///< line end
+   RAttrLine  fAttrLine{this, "line"};    ///<! line attributes
```

Comment:
IIRC the intention was to not line up member names in v7 anymore; you should be able to simply apply .clang_format.

Reply:
I apply clang-format for all headers in primitivesv7